dora/core/common/src/main/java/alluxio/uri/UfsUrl.java

```diff
@@ -223,11 +224,8 @@ public String getScheme() {
   /**
    * @return the authority of the {@link UfsUrl}
    */
-  public Optional<Authority> getAuthority() {
-    if (!mProto.hasAuthority()) {
-      return Optional.empty();
-    }
-    return Optional.of(Authority.fromString(mProto.getAuthority()));
+  public Authority getAuthority() {
```

Comment:
Why this change? IMO `Optional<Authority>` is more explicit than `NoAuthority`.

Reply:
Have replaced it with `Optional<Authority>`.